stage1/init/common/pod.go

```diff
@@ -1046,3 +1052,26 @@ func parseLinuxCapabilitiesSet(capSet types.LinuxCapabilitiesSet) []string {
 	}
 	return capsStr
 }
+
+func getAppNoNewPrivileges(isolators types.Isolators) (bool, error) {
+	noNewPrivs := ""
+
+	// TODO(sur): once https://github.com/appc/spec/pull/611 lands, bump appc spec, and use official appc types instead
```

Comment:
For the purpose of testing before appc/spec#611 lands, you could have a commit in this PR that patches Godeps/_workspace/src/github.com/appc/spec/ with the patch in appc/spec#611.